app/uk/gov/hmrc/apidocumentation/controllers/DocumentationController.scala

```diff
@@ -16,37 +16,39 @@
 
 package uk.gov.hmrc.apidocumentation.controllers
 
-import javax.inject.Inject
-
+import javax.inject.{Inject, Singleton}
 import org.joda.time.{DateTime, DateTimeZone}
 import play.api.Logger
-import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.i18n.I18nSupport
 import play.api.libs.json.Json
 import play.api.mvc._
 import uk.gov.hmrc.apidocumentation
-import uk.gov.hmrc.apidocumentation.ErrorHandler
 import uk.gov.hmrc.apidocumentation.config.ApplicationConfig
-import uk.gov.hmrc.apidocumentation.models.JsonFormatters._
 import uk.gov.hmrc.apidocumentation.models._
+import uk.gov.hmrc.apidocumentation.models.JsonFormatters._
 import uk.gov.hmrc.apidocumentation.services._
-import uk.gov.hmrc.apidocumentation.views
+import uk.gov.hmrc.apidocumentation.{views, ErrorHandler}
 import uk.gov.hmrc.apidocumentation.views.html._
+import uk.gov.hmrc.apidocumentation.views.html.include.{apiMain, main}
```

Comment:
These names could be better

Reply:
They're the names of the html templates - i'm not changing them.